api/src/flux.js

```diff
@@ -60,8 +60,8 @@ export const makeFluxNotifier = async (
   let currentRound;
 
   // Start a fresh query if we don't already have one.
-  const triggerQuery = async () => {
-    if (!querying) {
+  const triggerQuery = async (fromPollTimer) => {
+    if (!querying or fromPollTimer) {
```

Comment:
This seems to be a syntax error.

```suggestion
    if (!querying || fromPollTimer) {
```

Reply:
Ah yes sorry, this is because I initiated the PR from Github. Sorry about that, the rest of the code should be fine